### PR TITLE
i#3605: Support 64-bit literals for droption bytesize_t

### LINKED ISSUE #3605: droption bytesize_t does not support >32-bit literals (pre-suffix-expansion)
For an application i'm trying to collect traces with -trace_after_instr and -exit_after_tracing options. 
I'm trying to cut out the warm up phase, according to 'perf stat -e instructions -a' it took ~90B instructions. 
 perf stat -e instructions -a ^C
 Performance counter stats for 'system wide':
 92,106,941,277      instructions                                                
16.751698236 seconds time elapsed

But tracing tool fails due to count being too large. 

/root/DynamoRIO-x86_64-Linux-7.90.18019-0/bin64/drrun -t drcachesim -trace_after_instrs 90000000000 -exit_after_tracing 80000000000 -- /bin/sh /root/specpu2017/bin/runcpu --config=dummy-try2.cfg SPECspeed2017_int_base
Usage error: Option trace_after_instrs value out of range. 

Any suggestions on how does instructions count from perf tool correspond to how DynamoRIO is instruction counting.

### PR DESCRIPTION
**Pull request by benjaminrsherman**
Modifies convert_from_string() to read into a 64-bit value for the bytesize_t droptions.

Fixes #3605

### CODE REVIEW

File: ext/droption/droption.h [hunk 1]
```diff
@@ -584,9 +584,7 @@ droption_t<bytesize_t>::convert_from_string(const std::string s)
     std::string toparse = s;
     if (scale > 1)
         toparse = s.substr(0, s.size() - 1); // s.pop_back() only in C++11
-    // While the overall size is likely too large to be represented
-    // by a 32-bit integer, the prefix number is usually not.
-    int input = atoi(toparse.c_str());
+    long long input = atoll(toparse.c_str());
```
Comment (derekbruening) [inline ext/droption/droption.h]: Thank you for the pull request!

Could you add a test of a large number passed to a bytesize_t option to the suite/tests/client-interface/option_parse.dll.cpp test?  (Run "git grep val_sep2" to see which files are involved.)

Comment (benjaminrsherman) [reply ext/droption/droption.h]: Sorry for the delay.  Please let me know if I didn't add the test correctly.

**Comment by derekbruening** [comment]
run arm tests

**Comment by benjaminrsherman** [comment]
How do I do that?

**Comment by derekbruening** [comment]
"run arm tests" is a magic string that triggers running them when in a PR comment: i.e., I'm talking to a robot, not to you :)

**Comment by derekbruening** [comment]
#3882 will make the magic string need for non-core-devs go away

**Comment by derekbruening** [comment]
Thanks again for the patch.  Merged.

**Pull request merged**